temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
The `pdb` module is a debugging tool and should not be included in production code. If this is for debugging purposes, consider removing it before committing.
```
import pdb
```